components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/uuf-template-app/lib/modules/auth/auth.js

```diff
@@ -533,9 +530,9 @@ var module = {};
                     var ssoSessions = getSsoSessions();
                     ssoSessions[ssoSession.sessionId] = ssoSession;
                     if (ssoSession.sessionIndex != null || ssoSession.sessionIndex != 'undefined') {
```

Comment:
can simply use if (ssoSession.sessionIndex) ... -> this will check for null, undefined and also empty string